src/WebApiContrib.Core.Formatter.Csv/CsvFormatterOptions.cs

```diff
@@ -1,12 +1,14 @@
-﻿namespace WebApiContrib.Core.Formatter.Csv
+﻿using System.Text;
+
+namespace WebApiContrib.Core.Formatter.Csv
 {
     public class CsvFormatterOptions
     {
         public bool UseSingleLineHeaderInCsv { get; set; } = true;
 
         public string CsvDelimiter { get; set; } = ";";
 
-        public string Encoding { get; set; } = "ISO-8859-1";
+        public Encoding Encoding { get; set; } = Encoding.Default;

```

Comment:
why do we need a breaking change here? I think Encoding.GetEncoding is still there? https://github.com/dotnet/corefx/blob/master/src/Common/src/CoreLib/System/Text/Encoding.cs#L328

Reply:
if you want to add here a typed `Encoding` I'd suggest to `[Obsolete]` the other one for now instead of throwing it out, otherwise you break the semantic versioning

Reply:
Thanks for the review

The old/ or existing way has a bug which doesn't work, which I don't feel like chasing, because I should have used the typed way all the time, no idea why I did it like this before :( 

I suggest updating to 3.0.0 in version with this breaking change, good?